### ISSUE TITLE
request running on ps2 emulator 

### ISSUE THREAD

**Issue by dananothername** [anikkcah/PS2Linux#2]
cool if possible it would help in making a TAS

**Comment by anikkcah** [comment]
Yes I didn't thought of this aspect before. Share your progress if it runs on ps2 emulator.

**Comment by dananothername** [comment]
![Screenshot_2023-05-28-05-08-30-996_com discord](https://github.com/anikkcah/PS2Linux/assets/101902085/09fb8659-9922-4f8c-92cb-90d2ca550849)
i discussed this with pcsx2 dev
you can join their discord for help
